src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/burdenEstimateRepository/RetrieveBurdenEstimatesTests.kt

```diff
@@ -373,6 +375,63 @@ class RetrieveBurdenEstimatesTests : BurdenEstimateRepositoryTests()
         }
     }
 
+    @Test
+    fun `can getBurdenEstimateOutcomesSequence`()
```

Comment:
Can we add a few more tests here to test that data for the correct touchstone/scenario/set is returned? Want to avoid more bugs like that one where the responsibility set id and set id were the wrong way round :laughing: